modules/entity/src/sorted_state_adapter.ts

```diff
@@ -200,6 +210,7 @@ export function createSortedStateAdapter<T>(selectId: any, sort: any): any {
     updateOne: createStateOperator(updateOneMutably),
     upsertOne: createStateOperator(upsertOneMutably),
     addAll: createStateOperator(addAllMutably),
```

Comment:
I think you should rename `addAllMutalby` to `setAllMutable` and just point `addAll` and `setAll` to `createStateOperator(setAllMutably).`

Reply:
done